recipes/dunovo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+
+package:
+  name: dunovo
+  version: 0.6
+
+source:
+  fn: v0.6.tar.gz
+  url: https://github.com/galaxyproject/dunovo/archive/v0.6.tar.gz
+  sha256: a3cc378c22fcfc3bcaa11129f27f61bab5ff51cc9185320d53ece273b56e6214
+
+build:
+  number: 0
+
+requirements:
+  skip: True  # [py3k or osx]
+  build:
+    - make
+    - gcc
+
+  run:
+    - libgcc
+    - python >=2.6,<3
+    - argparse  # [py26]
```

Comment:
When constraining python like that, conda will not pin to the Python version used for the build. This can cause various technical issues. Please remove the constraint.